crates/biome_js_analyze/src/lint/nursery/no_static_element_interactions.rs

```diff
@@ -0,0 +1,353 @@
+// ref: https://github.com/jsx-eslint/jsx-ast-utils/blob/main/src/eventHandlers.js
+lazy_static::lazy_static! {
+    static ref INTERACTIVE_HANDLER_LIST: HashMap<&'static str, Vec<&'static str>> = {
+        let mut m = HashMap::new();
+        m.insert("clipboard", vec!["onCopy", "onCut", "onPaste"]);
+        m.insert("composition", vec!["onCompositionEnd", "onCompositionStart", "onCompositionUpdate"]);
+        m.insert("keyboard", vec!["onKeyDown", "onKeyPress", "onKeyUp"]);
+        m.insert("focus", vec!["onFocus", "onBlur"]);
+        m.insert("form", vec!["onChange", "onInput", "onSubmit"]);
+        m.insert("mouse", vec![
+            "onClick", "onContextMenu", "onDblClick", "onDoubleClick", "onDrag", "onDragEnd",
+            "onDragEnter", "onDragExit", "onDragLeave", "onDragOver", "onDragStart", "onDrop",
+            "onMouseDown", "onMouseEnter", "onMouseLeave", "onMouseMove", "onMouseOut",
+            "onMouseOver", "onMouseUp"
+        ]);
+        m.insert("selection", vec!["onSelect"]);
+        m.insert("touch", vec!["onTouchCancel", "onTouchEnd", "onTouchMove", "onTouchStart"]);
+        m.insert("ui", vec!["onScroll"]);
+        m.insert("wheel", vec!["onWheel"]);
+        m.insert("media", vec![
+            "onAbort", "onCanPlay", "onCanPlayThrough", "onDurationChange", "onEmptied",
+            "onEncrypted", "onEnded", "onError", "onLoadedData", "onLoadedMetadata", "onLoadStart",
+            "onPause", "onPlay", "onPlaying", "onProgress", "onRateChange", "onSeeked", "onSeeking",
+            "onStalled", "onSuspend", "onTimeUpdate", "onVolumeChange", "onWaiting"
+        ]);
+        m.insert("image", vec!["onLoad", "onError"]);
+        m.insert("animation", vec!["onAnimationStart", "onAnimationEnd", "onAnimationIteration"]);
+        m.insert("transition", vec!["onTransitionEnd"]);
+        m
+    };
+}
+
+impl Rule for NoStaticElementInteractions {
+    type Query = Aria<AnyJsxElement>;
+    type State = ();
+    type Signals = Option<Self::State>;
+    type Options = ();
+
+    fn run(ctx: &RuleContext<Self>) -> Self::Signals {
+        let node = ctx.query();
+        let aria_roles = ctx.aria_roles();
+        let element_name = node.name().ok()?.as_jsx_name()?.value_token().ok()?;
+        let attributes = extract_attrs(&node.attributes());
+        let elm = element_name.text_trimmed();
+
+        if let Some(attributes_ref) = attributes.as_ref() {
+            if !is_interactive_handler_present(attributes_ref) {
+                return None;
+            }
+        } else {
+            return None;
+        }
+
+        let default_attributes = FxHashMap::default();
+        let attributes_ref = attributes.as_ref().unwrap_or(&default_attributes);
+
+        let attributes_option = if !attributes_ref.is_empty() {
+            Some(attributes_ref.clone())
+        } else {
+            None
+        };
```

Comment:
it seems that we don't need this process.

Reply:
@unvalley 
fixed in this commit 679fbabd54fe72a0c8053a9b5163b4cfec9a496f